config/config.example.php

```diff
@@ -1,3 +1,4 @@
 <?php
 $matomoUrl = 'http://localhost';
 $matomoIdSite = '1';
+$matomoTagManagerContainer = "ZNBuKNcy";
```

Comment:
could maybe add this to the documentation in https://github.com/matomo-org/test-examples/blob/main/README.md#configuration ?

Add the beginning of the file could actually add a comment like below?

```php
// WHEN ADDING NEW VARIABLES PLEASE DOCUMENT THEM IN THE README
```